ui/src/app/pages/data/concept/survey-details.tsx

```diff
@@ -82,11 +82,11 @@ export const SurveyDetails = withCurrentWorkspace()(
 
     async loadSurveyDetails() {
       try {
-        const {workspace, surveyName} = this.props;
-        const surveys = await conceptsApi().getSurveyQuestions(
-          workspace.namespace, workspace.id, surveyName);
+        const {surveyName, workspace} = this.props;
+        const request = {standardConceptFilter: StandardConceptFilter.ALLCONCEPTS, surveyName};
+        const surveys = await conceptsApi().searchSurveys(workspace.namespace, workspace.id, request);
```

Comment:
This new api call brings questions back in same order as CB.

Reply:
Ok, cool. That helps when we're ready to add numbers to the rows to match data browser.